server/container_start.go

```diff
@@ -57,6 +57,13 @@ func (s *Server) StartContainer(ctx context.Context, req *types.StartContainerRe
 	if err := s.Runtime().StartContainer(ctx, c); err != nil {
 		return fmt.Errorf("failed to start container %s: %v", c.ID(), err)
 	}
+	if s.config.EventedPLEG {
+		if err := s.Runtime().UpdateContainerStatus(ctx, c); err != nil {
+			return fmt.Errorf("failed to update container status %s: %v", c.ID(), err)
+		}
```

Comment:
I'm wondering why we have to update the container status? Before throwing an event?

Reply:
This [link](https://kubernetes.slack.com/archives/C03HA69L6R1/p1658751854843449) actually summarizes the issue and the fix looks to be updating the pod status prior to the generation of an event.

cc: @harche